exporter/otlphttpexporter/otlp.go

```diff
@@ -80,7 +82,15 @@ func (e *exporterImp) pushTraceData(ctx context.Context, traces pdata.Traces) (i
 		return traces.SpanCount(), consumererror.Permanent(err)
 	}
 
-	err = e.export(ctx, e.tracesURL, request)
+	const TOKEN = "$TOKEN"
+	if strings.Contains(e.tracesURL, TOKEN) {
+		var c, _ = client.FromContext(ctx)
```

Comment:
We should extract the error and handle it gracefully (which is a good question on what should happen in such case). We could e.g. throw an error if no token is available in client context